Makefile

```diff
@@ -31,7 +31,7 @@ DAEMON_CONTAINER := ${DOCKER_PREGISTRY_URL}/${DAEMON_IMAGE}:${DAEMON_TAG}
 all: clean build create-app-image deploy-skill
 
 create-app-image:
-	docker build --build-arg base_img=c12e/profiles-sdk:1.2.3-gec36747 -t ${DOCKER_IMAGE}:${TAG} -f ./main-app/build/resources/main/Dockerfile ./main-app/build
+	docker build --build-arg base_img=c12e/profiles-sdk:1.2.9-gb19b2b3 -t ${DOCKER_IMAGE}:${TAG} -f ./main-app/build/resources/main/Dockerfile ./main-app/build
```

Comment:
Should we update to latest version `c12e/profiles-sdk:1.3.1-gb3c97be` of docker image?